test/Puzzle-test.js

```diff
@@ -1,3 +1,40 @@
 const chai = require('chai');
 const expect = chai.expect;
-const Puzzle = require('../js/Puzzle.js');
\ No newline at end of file
+const Puzzle = require('../js/Puzzle.js');
+
+describe('Puzzle', function () {
+  it('should check to see if the puzzle has been completed', function () {
+    var puzzle = new Puzzle();
+    expect(puzzle.puzzleCompleted).to.equal(false);
```

Comment:
Can you please explain the 2 assertions? I can't understand why one is false and the next one is true, but nothing has changed.